docs/install/containers/recipes/index.md

```diff
@@ -122,12 +122,15 @@ It will no longer output the access log, but will continue to output the event l
 
 ## Pack the ZODB
 
+```{versionadded} Plone 6.0.15
```

Comment:
and 6.1.1

Reply:
Done in https://github.com/plone/documentation/pull/1899/commits/031414e5d5862cf5926661f58f38a9a09714c8ee